### PR TITLE
(WIP) Different backbones support (resnetXX, mobilenet_v2)

### PR DESCRIPTION
**Pull request by kyrylogr**

### CODE REVIEW

File: models/backbones/abstract_backbone.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+import torch.nn as nn
+
+
+class AbstractBackbone(nn.Module, ABC):
+
+    @abstractmethod
+    def forward(self, x):
```
Comment (bubanoid) [inline models/backbones/abstract_backbone.py:9]: Due to the the implementation of the  `nn.Module` class there is no need to define abstract class.

```Python
class Module:
    ...
    forward: Callable[..., Any] = _forward_unimplemented
    ...
    def _forward_unimplemented(self, *input: Any) -> None:
            raise NotImplementedError('smth...')
```

File: training/train.py
Comment (bubanoid) [inline training/train.py:1]: Please remove your changes of this file

**Comment by macoidk** [comment]
add please imagenet and convnext large backbones

File: models/backbones/abstract_backbone.py (hunk 1 above)
Comment (kyrylogr) [reply models/backbones/abstract_backbone.py:9]: removed ABC from baseclasses

**Pull request closed without merging**